src/App.js

```diff
@@ -1,7 +1,12 @@
 import "./App.css";
-
+import Card from "./Card/Card";
+import data from "../src/Card/CardData";
 function App() {
-  return <div className="App"></div>;
+  const travelCountry = data.map((item) => {
+    return <Card key={item.id} {...item} />;
+  });
+
+  return <div className="App">{travelCountry}</div>;
```

Comment:
- [ ] use object destructuring here in order to avoid object.chaining

Reply:
Thank you for this comment. It was new method that I learned. Used object destructuring.